packages/scripts/src/verifiedAnswers/parseVerifiedAnswersYaml.ts

```diff
@@ -0,0 +1,31 @@
+import { parse } from "yaml";
+import { z } from "zod";
+
+export const referenceSchema = z.object({
+  uri: z.string(),
+  title: z.optional(z.string()),
+});
+
+export type Reference = z.infer<typeof referenceSchema>;
```

Comment:
Minor - we could base these on the `rag-core` definition. https://github.com/mongodb/chatbot/blob/main/packages/mongodb-rag-core/src/services/conversations.ts

If we want title to be optional I think you can extend & override like this:

```ts
const existingSchema = z.object({
  fieldName: z.string()
});

const newSchema = existingSchema.extend({
  fieldName: z.string().optional()
});
```